ci/setup_env.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash -e
+
+
+# edit the locale file if needed
+function edit_init()
+{
```

Comment:
I don't see the value of having this in a function, as it's called just once and immediately after defining it. Can you simply run this code directly?